ansible/roles/stunner/tasks/main.yml

```diff
@@ -32,11 +32,12 @@
     state: "{{ stunner_absent_or_present }}"
 
 - name: Configure stunner
+  when: stunner_absent_or_present == "present"
   kubernetes.core.k8s:
     template: "stunner-configuration.yml"
-    state: "{{ stunner_absent_or_present }}"
```

Comment:
Why this?
    state: "{{ stunner_absent_or_present }}"
is the same as 
    when: stunner_absent_or_present == "present"

Reply:
because of an error :
```
fatal: [localhost]: FAILED! => {"changed": false, "msg": "Failed to find exact match for gateway.networking.k8s.io/v1.GatewayClass by [kind, name, singularName, shortNames]"}
```
I found it easier to bypass it this way.

Reply:
oki